pkgs/development/python-modules/pyatv/default.nix

```
@@ -31,6 +36,8 @@ buildPythonPackage rec {
     protobuf
     cryptography
     netifaces
+    zeroconf
+    pytestCheckHook
```

```suggestion
    pytestCheckHook
```
this should be in checkInputs